pkgs/top-level/python-packages.nix

```diff
@@ -4660,6 +4660,11 @@ in {
     inherit (pkgs) pkgconfig;
   };
 
+  wxPython40 = callPackage ../development/python-modules/wxPython/4.0.nix {
```

Comment:
AFAIK the preferred attribute naming convention for wxPython version 4.0.x is `wxPython_4_0`

Reply:
correct, nixops for example has nixops_1_6_1 for it's last release that supports azure.

Reply:
inconsistent with wxPython30 but sounds good

Reply:
just because something happened; doesn't mean that the action was correct initially, or that the standards haven't changed since then :)

You can only do the best you can, with what you're given at the time.